### PR TITLE
Add .editorconfig

### PR DESCRIPTION
**Pull request by ryonakano**

### CODE REVIEW

**Review by jeremypw — APPROVED** [review]

**Comment by ryonakano** [comment]
Looks like merging is blocked by the ghost "build" CI which marked as Required. Maybe this PR can be merged only after some repository admin removes this CI from the branch protection rule in the repository setting.

File: .editorconfig
Comment (jeremypw) [reply .editorconfig:11]: Most if not all other elementary repositories have 80 as the editorconfig max_line_length.  I think this is so Code puts the line width *guide* at 80 characters  I presume this is so code lines (not including comments?) should endeavour to be shorter than this for readability whereas 120 is a hard limit that triggers a lint error unless overridden.

If I am wrong and this needs to be updated then it will need to be done in many other repositories.

**Comment by jeremypw** [comment]
@danirabbit Are you able to amend the branch protection rules to allow CI to pass? Or is it blocked for a reason?

**Pull request merged**
